src/components/TopBar/ExportButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useCallback } from 'react';
+import { observer } from 'mobx-react';
+
+import { Icon, Button } from '@blueprintjs/core';
+
+import css from './styles.scss';
+
+export interface Props {
+  onExport?: () => void;
+}
+
+export const ExportButton = observer(function ExportButton(props: Props) {
```

Comment:
```suggestion
export const ExportButton = function ExportButton(props: Props) {
```
you are not using any Mobx elements in this component, so no need to wrap it in `observer`.